api/deploy.go

```diff
@@ -88,6 +88,9 @@ func deploy(w http.ResponseWriter, r *http.Request, t auth.Token) (err error) {
 	if err != nil {
 		return &tsuruErrors.HTTP{Code: http.StatusNotFound, Message: err.Error()}
 	}
+	if (instance.GetPlatform() == "") && (origin != "image") {
+		return &tsuruErrors.HTTP{Code: http.StatusUnauthorized, Message: "can't deploy app without platform, if It's not a docker image"}
```

Comment:
Just a tiny detail: I would remove `docker` from this error message. And downcase the word `It's`